.travis.yml

```diff
@@ -1,24 +1,19 @@
 dist: bionic
-go:
-  - "1.12"
-go_import_path: github.com/algorand/go-algorand
-language: go
 sudo: required
+language: minimal
 
 # Don't build tags
 if: tag IS blank
```

Comment:
If we're getting rid of the `deploy` stage we can get rid of this. The old deploy script had a dependency on the branch name (`rel/stable`, `rel/nightly`, etc) which is why we couldn't build tags.

Reply:
This may still be required because I'm actually just calling the script that the deploy stage used as a make target. Later on we're planning to write a mule task that handles the uploads on it's own, so maybe then will be a better time to remove this

```
archive:
	PATH=$(SRCPATH)/tmp/node_pkgs/$(OS_TYPE)/$(ARCH)/dev/$(OS_TYPE)-$(ARCH)/bin:$${PATH} \
	scripts/upload_version.sh $(BUILDCHANNEL) $(SRCPATH)/tmp/node_pkgs/$(OS_TYPE)/$(ARCH) $(S3_RELEASE_BUCKET)
```